README.md

```diff
@@ -239,6 +239,27 @@ or
   or earlier and seeing ambiguous module errors or other mysterious
   failures, try manually unsetting `GHC_ENVIRONMENT` before invoking
   `doctest`.
+  
+ * If you are on Nix. `doctest` will not pick up your version of GHC if you
+   don't point it towards it, and therefore will result in "cannot satisfy -package-id" errors.
+   You will need to set the following variables within your environment in order
+   for doctest to pick up your GHC, with /lib/ghc-8.10.7 changed to your version of ghc:
+```nix
```

Comment:
How to test this?  I put the contexts into a `flake.nix` file and then ran 
```
$ nix --extra-experimental-features nix-command --extra-experimental-features flakes flake check .
error: syntax error, unexpected ELLIPSIS

       at /nix/store/n877x4s8rbk3b1m4vni1x8w5i6x2n5x1-source/flake.nix:3:65:

            2| let
            3|   myHaskell = (pkgs.haskellPackages.ghcWithHoogle (p: with p; [ ... ]));
             |                                                                 ^
            4| in
```

I don't know `nix`.  Anyone suggestions for a reviewer of this PR?